CMakeLists.txt

```diff
@@ -56,8 +56,8 @@ set(GZ_COMMON_VER ${gz-common6_VERSION_MAJOR})
 
 #--------------------------------------
 # Find gz-transport
-gz_find_package(gz-transport14 REQUIRED)
-set(GZ_TRANSPORT_VER ${gz-transport14_VERSION_MAJOR})
+gz_find_package(gz-transport REQUIRED)
+set(GZ_TRANSPORT_VER ${gz-transport15_VERSION_MAJOR})
```

Comment:
is the `GZ_TRANSPORT_VER` var still needed?

Reply:
no, thanks for the reminder. I manually removed those variables in follow-ups to https://github.com/gazebosim/gz-gui/pull/670, but it wasn't easy to automate

I'll remove it manually

Reply:
removed in https://github.com/gazebosim/gz-sensors/pull/515/commits/a9e8c29ec324746b407e0ff054ae73d83041d5e8